File: PoweredSoft.Music.String.Core/IStringInstrumentChord.cs
using System.Collections.Generic;
using PoweredSoft.Music.String.Core;
using PoweredSoft.Music.Theory.Core;


namespace PoweredSoft.Music.String.Core
{
    public interface IStringInstrumentChord
    {
        IChord Chord { get; set; }
        IList<IStringInstrumentChordPossibility> ChordPossibilities { get; set; } 
    }
}

File: PoweredSoft.Music.Web/spa/src/models/IStringInstrumentChord.ts
import { IChordDefinition } from './IChordDefinition';
import { IStringInstrumentChordPossibility } from './IStringInstrumentChordPossibility';
export interface IStringInstrumentChord {
    chord: IChordDefinition;
    chordPossibilities: IStringInstrumentChordPossibility[];
}
